app/work/page.tsx

```diff
@@ -1,8 +1,188 @@
-import React from 'react'
+"use client";
+
+import { motion } from "framer-motion";
+import React, { useState } from "react";
+
+import { Swiper, SwiperSlide } from "swiper/react"
+import { Swiper as SwiperType } from "swiper"
+import "swiper/css"
+
+import { 
+  BsArrowUpRight,
+  BsGithub
+} from 'react-icons/bs'
+
+import { 
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger
+} from "@/components/ui/tooltip";
+
+import Link from "next/link";
+import Image from "next/image";
+import WorkSliderBtn from "@/components/WorkSliderBtn";
+
+const projects = [
```

Comment:
Since this is a relatively large data object, maybe try setting it in a JSON file outside the code.

Reply:
@emkkur Can I skip this for now? I will do this in the end as part of optimization.